synapse/replication/slave/storage/_slaved_id_tracker.py

```diff
@@ -36,17 +36,7 @@ def advance(self, instance_name: Optional[str], new_id: int):
         self._current = (max if self.step > 0 else min)(self._current, new_id)
 
     def get_current_token(self) -> int:
-        """
-
-        Returns:
-            int
-        """
         return self._current
 
     def get_current_token_for_writer(self, instance_name: str) -> int:
-        """Returns the position of the given writer.
-
-        For streams with single writers this is equivalent to
-        `get_current_token`.
-        """
```

Comment:
Removing type hints in documentation is alright, but this is actual function documentation, i dont think this should be removed.

Reply:
As mentioned here (https://github.com/matrix-org/synapse/pull/11411#discussion_r755131187), the docstrings have been moved to `AbstractStreamIdTracker`.